### PR TITLE
feat:Introduce support for Int64 and UInt64 for protobuf

### LINKED ISSUE #11: ✨ Add support for int64
## Is your feature request related to a problem? Please describe.

While [explicitly not supported by JTD](https://www.rfc-editor.org/rfc/rfc8927#section-a.1), protobuf [does support `int64` and `uint64` types](https://developers.google.com/protocol-buffers/docs/proto3). Since the usage of JTD here is only for schema management, the reasons given in the JTD RFC don't apply, and we can safely extend `jtd_to_proto` to allow for `int64` and `uint64`.

## Describe the solution you'd like

Add `int64` and `uint64` to [the JTD to proto type map](https://github.com/IBM/jtd-to-proto/blob/main/jtd_to_proto/jtd_to_proto.py#L53)

## Describe alternatives you've considered

Stick with `int32`. The reason to not do this is that on 64-bit machines, python's built in `int` is an `int64`, so supporting an `int64` in `jtd_to_proto` will allow us to avoid higher-order errors with mapping `int` -> `int64` -> `TYPE_INT64`.

### PR DESCRIPTION
**Pull request by viveknshah**
Fixes #11 

Verified Tests locally:

image

### CODE REVIEW

**Comment by viveknshah** [comment]
@gabe-l-hart Not quite sure but I figured this was the expected outcome of the issue? Let me know if I misunderstood.

File: jtd_to_proto/jtd_to_proto.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def _to_upper_camel(snake_str: str) -> str:
     "uint8": _descriptor.FieldDescriptor.TYPE_UINT32,
     "int32": _descriptor.FieldDescriptor.TYPE_INT32,
     "uint32": _descriptor.FieldDescriptor.TYPE_UINT32,
+    "int64": _descriptor.FieldDescriptor.TYPE_INT64,
+     "uint64": _descriptor.FieldDescriptor.TYPE_UINT64,
```
Comment (gabe-l-hart) [inline jtd_to_proto/jtd_to_proto.py:53]: NIT: I think there's an extra space on this line

**Review by gabe-l-hart — CHANGES_REQUESTED** [review]
Thanks for the contribution @viveknshah! Minor NIT on the spacing which you can fix by running `make fmt` in the base of the repo. Also, IBM's org enforces signed commits, so you can do the following to sign: `git fetch origin && git rebase origin/main --signoff`.

Finally, it would be great to add a quick unit test to explicitly verify that the new types are supported. We've still got 100% coverage as is since the dict is declared, but it would be great to ensure that nothing breaks in the JTD validation. The test would look a lot like [this one](https://github.com/IBM/jtd-to-proto/pull/6/files#diff-93d9a184f67d4a3847d6e06f9fb74e45afcf4404e74320076ec07a039071c0dfR597).

**Comment by viveknshah** [comment]
> Thanks for the contribution @viveknshah! Minor NIT on the spacing which you can fix by running `make fmt` in the base of the repo. Also, IBM's org enforces signed commits, so you can do the following to sign: `git fetch origin && git rebase origin/main --signoff`.
> 
> Finally, it would be great to add a quick unit test to explicitly verify that the new types are supported. We've still got 100% coverage as is since the dict is declared, but it would be great to ensure that nothing breaks in the JTD validation. The test would look a lot like [this one](https://github.com/IBM/jtd-to-proto/pull/6/files#diff-93d9a184f67d4a3847d6e06f9fb74e45afcf4404e74320076ec07a039071c0dfR597).

Thank you for your patient feedback. Have made the appropriate formating flows, verified the added tests and followed the signed commit command.

**Pull request closed without merging**